docs/src/pages/components/alert/react.mdx

```diff
@@ -126,6 +127,10 @@ Use the `onDismiss` prop to pass a function that will run when the Alert is dism
 
 ## CSS Styling
 
+### Target classes
+
+<ComponentClassTable componentName="Alert" />
```

Comment:
Ah this is so much easier to document!

Reply:
Totally! Thanks for doing this, @jacoblogan